README.md

```diff
@@ -8,7 +8,19 @@ Attualmente è composto da:
 - [Note dei meeting degli Ambassador](./meetings/)
 - [Eventi al quale il progetto ha partecipato](./events/)
 
-# Contribuire
+## Struttura file e cartelle
+
+Il repository è strutturato in cartelle, ognuna delle quali contiene un file `README.md` che ne descrive il contenuto.
+
+Questa è lista dettagliata delle cartelle
+
+- `events`: contiene le note degli eventi a cui il progetto ha partecipato, organizzate per data, la quantità si consiglia di creare delle sottocartelle per working group e per anno
```

Comment:
```suggestion
- `events`: Contiene le note degli eventi a cui il progetto ha partecipato, ordinate per data.
```

Suggerisco di rimuovere il riferimento al WG anche perché agli eventi non partecipa un WG ma gli ambassador in toto.. O non ho capito io?